.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
**question:** why has this been added? x-dash use [volta](https://docs.volta.sh/guide/) for node management (see [here in package.json](https://github.com/Financial-Times/x-dash/pull/713/files#diff-92213008ca2c99f2c62c106f3d1831faf0ae65d7309b4d54cb10826f39c04443L40)) for node management. FYI for context in Customer Products nvm used to be the default but now most projects use volta so if we can stick with that then lets.

Reply:
ok